addon-test-support/@ember/test-helpers/setup-application-context.js

```diff
@@ -19,7 +19,12 @@ export function visit() {
       return owner.visit(...arguments);
     })
     .then(() => {
-      context.element = document.querySelector('#ember-testing > .ember-view');
+      // eslint-disable-next-line
+      if (EmberENV.FEATURES['ember-glimmer-remove-application-template-wrapper']) {
```

Comment:
This might work for now, but isn't a good long term solution (once it moves to beta this will be falsey).

I think what we need to do is render an empty template, detect if there are any `ember-view`s and then memoize (so we only have to do the extra rendering one time for the entire suite)/

Reply:
@cibernox - Now that this has settled, we can use `EmberENV._APPLICATION_TEMPLATE_WRAPPER`. A `true` value means we need `#ember-testing > .ember-view` and a `false` value means we need `#ember-testing`.

Reply:
I'll update it.